website/docs/docs/build/unit-tests.md

```diff
@@ -7,11 +7,11 @@ id: "unit-tests"
 keywords:
   - unit test, unit tests, unit testing, dag
 ---
-:::note closed beta
+:::note beta
 
-Support for unit testing dbt models is available to dbt Cloud customers who have chosen to [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version). This is currently in closed beta, starting with select customers and rolling out to wider availability through February and March.
+Support for unit testing dbt models is available to dbt Cloud customers who have chosen to ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version). This feature is currently available in beta, and supports all dbt-Labs-maintained adapters, with more adapter availability rolling out through March.
```

Comment:
"Rolling out through March" Will new adapter availability be trickling out or do we plan to release on a single day? The docs waver between "through March" "full availability by end of march" and "widely available by end of march" and if we're tying all of these features together, it might benefit us and our customers to use consistent language across these notices to avoid confusion.

Reply:
These will be released adapter-by-adapter cc: @jtcohen6